src/aap_eda/analytics/analytics_collectors.py

```diff
@@ -40,6 +41,40 @@ def config(**kwargs) -> dict:
     }
 
 
+@register(
+    "jobs_stats",
+    "1.0",
+    description="Stats data for jobs",
+)
+def jobs_stats(since: datetime, full_path: str, until: datetime, **kwargs):
+    stats = {}
+    audit_actions = _get_audit_action_qs(since, until)
+
+    if not bool(audit_actions):
+        return stats
+
+    for action in audit_actions.all():
+        job_type, job_id, install_uuid = extract_job_details(action.url)
```

Comment:
Can action.url be empty? If the action is not a controller job, it does not have an url.

Reply:
the `action.url` can only be an empty string in its model definition, which is handled properly.